src/ontology/mondo-edit.obo

```diff
@@ -131286,8 +131286,7 @@ xref: DOID:0080344 {source="MONDO:equivalentTo"}
 xref: GARD:0002071 {source="MONDO:equivalentTo"}
 xref: ICD10:Q87.8 {source="ORDO:1997/attributed", source="ORDO:1997/ntbt", source="Orphanet:1997"}
 xref: MESH:C536188 {source="MONDO:equivalentTo", source="MONDO:ontobio"}
-xref: OMIM:119580 {source="ORDO:1997/e", source="GARD:0002071", source="Orphanet:1997"}
```

Comment:
Why did this go away? It correctly relfects the case that ordo declares equivalence

https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http://www.orpha.net/ORDO/Orphanet_1997

note we don't declare equivalence

Reply:
Did you see my commit message?

remove xrefs from 'familial avascular necrosis of femoral head'

these come from Orphanet, they xref the more specfic OMIM classes, because they only have the general class, but we have the more specific OMIM subclasses.

Does this not properly address the issue?

See ticket: https://github.com/monarch-initiative/mondo/issues/1145

Reply:
we have been keeping these but we don't add any semantics to the xref

Reply:
ah, okay. I added that dbxref back